### DISCUSSION TITLE
Pull Shark Badge Archievement Issue

### DISCUSSION THREAD

**Discussion by codev911** [community/community#18334] [Projects and Issues]
Hi Github team, i just received wrong archievement here. My pull has been closed and not merged by openzeppelin team but i got this achievement, please fix it, thanks :)

![wrong-archievement](https://user-images.githubusercontent.com/53482087/173177510-2fb48e75-ad3f-445a-8962-20172d333711.png)

**Comment by melvincwng** [comment]
Hi @codev911 , as of right now, I don't think there is a way to remove a specific achievement from your Github Profile page

Couple of options that I can think of for you right now:
1. Go to 'Settings' > 'Public Profile' > Scroll down to 'Profile Settings' > Uncheck the option 'Show Achievements on my profile'. (This will hide your achievements in your Github profile page)
2. Wait till a Github Admin reverts that achievement manually from your profile.

Hope this works, all the best

**Comment by Nishant1500** [comment]
You get that badge for a certain amount of pull request that got merged in your whole lifetime. You have that badge probably because you have 2 or more PRs that have been merged. It doesn't depend on one pull request.

**Comment by zorn-v** [comment]
Also seems this badge does not work for merged pull requests that created via "Edit file" github feature.

**Reply by Nishant1500** [reply]
Huh, it should work, probably you are wrong.

**Reply by zorn-v** [reply]
https://github.com/websanova/vue-auth/pull/689 does not count for example.
To me it does not matter, but bug report )

**Reply by Nishant1500** [reply]
How much merged PRs you have right now?

**Reply by zorn-v** [reply]
![изображение](https://user-images.githubusercontent.com/12619075/173192543-da5f33b6-863f-4941-b8d3-3d696913dc16.png)

**Reply by zorn-v** [reply]
Ah, seems I got. It not for every merged PR, but like 2nd, 16th etc.

**Comment by alizain-c** [comment]
Hi @codev911! Hope all is well! 
 So the "Pull shark" achievement in itself counts all the PRs that you've made all throughout the existence of your GitHub account. If a PR was merged by any chances in 2018 for instance, and you opened it any time later, you will get that achievement. "YOLO" is similar IMO, you get that if you successfully merge PR. 
 
 Hence, it's not an issue or something to worry about. I hope it helps!

**Comment by Sulagna-Dutta-Roy** [comment]
No, this badge is relevant to you. Probably you merged 2 or many PRs. So relax 

You can hide your achievement by doing this from settings->unchecked->update preferences. 
Maybe this helpful to you.
https://github.com/Schweinepriester/github-profile-achievements

![Github](https://user-images.githubusercontent.com/72568715/173202608-3981477b-a7be-4664-b29c-76293d85e759.PNG)

**Comment by andrewdimmer** [comment]
Looking at it, it appears that the "Pull Shark" was retro-actively awarded on the date of the pull request creation, and not the date of the merge (or, closure in this case). I'm wondering if maybe that was data backfill issue, or if that is an actual issue with the badge criteria itself...

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
